roadmap/implementers-guide/src/types/disputes.md

```diff
@@ -33,8 +33,8 @@ Kinds of dispute statements. Each of these can be combined with a candidate hash
 ```rust
 enum ValidDisputeStatementKind {
     Explicit,
-    BackingSeconded,
-    BackingValid,
+    BackingSeconded(Hash),
```

Comment:
I'd expect this to be used with key value store, where the key is the hash, otherwise all other variants would need the `Hash` as well?

Reply:
I don't know what a key-value store has to do with this, but in the case of `BackingSeconded` and `BackingValid`, the hash is the 'inclusion parent' or relay-parent referenced by the `SigningContext`. neither explicit dispute statements nor approval votes require the `SigningContext` as they have no need to be pinned to any specific point in the chain.